packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/NavigationItem.tsx

```diff
@@ -52,6 +61,18 @@ const Container = styled(NavigationItemBase)<{
                       }
                   }
               `}
+    ${({ $isRounded }) =>
+        $isRounded &&
+        css`
+            border-radius: ${borders.radii.full};
+            padding: ${spacingsPx.xs} ${spacingsPx.md};
+        `}
+    ${({ $typographyStyle }) =>
```

Comment:
I think this can be removed and the Translation element wrapped in Paragraph. And the Count element can also be removed, and replaced with Text/Paragraph with variant="tertiary".

Reply:
Idk about removing typographyStyle because I still need to have a different gap, otherwise it looks weird with smaller text

Reply:
The Paragraph has margin prop, you can set the values conditionally.